.eslintrc.js

```diff
@@ -23,10 +23,35 @@ module.exports = {
   plugins: [
     'react',
 		'@typescript-eslint',
+	'import',
   ],
+  "settings": {
+    "import/parsers": {
+      "@typescript-eslint/parser": [".ts", ".tsx"]
+    },
+    "import/resolver": {
+      "typescript": {
+        "alwaysTryTypes": true // always try to resolve types under `<root>@types` directory even it doesn't contain any source code, like `@types/unist`
+      },
+	},
+	"import/external-module-folders": ["node_modules"],
+	"import/internal-regex": "^@*/"
+  },
   rules: {
 		// These rules are in alphabetical order, minus any symbols (i.e. @typescript goes under T).  Please preserve this when adding.
+		'array-bracket-newline': [
```

Comment:
Did you use tabs here to retain consistency within this block?

Reply:
the entire file's formatting was inconsistent... ironic

i couldn't figure out how to apply the rules to this file, and i didn't want to correct styling on it yet. just wanted to get the right rules down